src/README.md.jinja

```diff
@@ -12,7 +12,12 @@
     {{- version | replace(".","-") -}}
 {%- endmacro -%}
 [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/{{ repo_id(repo_slug) }}/{{ odoo_version }}.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-{{ repo_slug }}-{{ repo_id(repo_slug) }})
+{%- if ci == "Travis" %}
 [![Build Status](https://travis-ci.com/OCA/{{ repo_slug }}.svg?branch={{ odoo_version }})](https://travis-ci.com/OCA/{{ repo_slug }})
+{%- else %}
+[![Pre-commit Status](https://github.com/OCA/{{ repo_slug }}/actions/workflows/pre-commit.yml/badge.svg?branch={{ odoo_version }})](https://github.com/OCA/{{ repo_slug }}/actions/workflows/pre-commit.yml?branch={{ odoo_version }})
+[![Build Status](https://github.com/OCA/{{ repo_slug }}/actions/workflows/test.yml/badge.svg?branch={{ odoo_version }})](https://github.com/OCA/{{ repo_slug }}/actions/workflows/test.yml?branch={{ odoo_version }})
```

Comment:
Actually, the little badge I have written doesn't get printed in the README, as you can see in https://github.com/OCA/l10n-spain. The Github badges just print the workflow name, so there isn't much we can do :man_shrugging:

Reply:
Ah, I see. Ok then. Maybe we should change the names in the github workflows?
And perhaps rename the test jobs (because they are both get named as "test Odoo addons" and it would be clearer if they'd mention Odoo and OCB, so "Test with Odoo" and "Test with OCB")

Reply:
That would be good, yes. However, the current name comes from [here](https://github.com/OCA/oca-addons-repo-template/blob/master/src/.github/workflows/%7B%25%20if%20ci%20==%20'GitHub'%20%25%7Dtest.yml%7B%25%20endif%20%25%7D.jinja#L1) so the only way would be splitting the 2 tests (OCB and Odoo) in 2 different files and 2 different workflows...